src/views/BusinessCase/index.tsx

```diff
@@ -151,7 +136,7 @@ export const BusinessCase = () => {
         </div>
         {businessCase.id && (
           <Switch>
-            <SecureRoute
+            <Route
```

Comment:
I'm curious about this change. Are these routes now accessible without logging in?

Reply:
Great call out. As you might remember, we had some warnings/errors with the export of `SecureRoute`. That’s why I changed it. 

Because the “base” route in `App/index.tsx` is still `SecureRoute` it will still be protected. I have a suspicion that there’s issues with rendering nested `SecureRoute`s, but haven’t done any investigation to confirm that.